pages/volunteer/Volunteer.js

```diff
@@ -0,0 +1,122 @@
+      designSlideQuotes.push(designerProfile.professionalQuote);
+      designSlideNames.push(designerProfile.fullName);
+      designSlideJobTitles.push(designerProfile.jobTitle);
+    });
+    $scope.designSlideQuotes = designSlideQuotes;
+    $scope.designSlideNames = designSlideNames;
+    $scope.designSlideJobTitles = designSlideJobTitles;
+
+    var developmentSlideImages = [];
+    var developmentSlideQuotes = [];
+    var developmentSlideNames = [];
+    var developmentSlideJobTitles = [];
+    VOLUNTEER_INFO.development.forEach(function(developerProfile) {
+      developmentSlideImages.push(developerProfile.profilePictureImageFilename);
+      developmentSlideQuotes.push(developerProfile.professionalQuote);
+      developmentSlideNames.push(developerProfile.fullName);
+      developmentSlideJobTitles.push(developerProfile.jobTitle);
+    });
+    $scope.developmentSlideQuotes = developmentSlideQuotes;
+    $scope.developmentSlideNames = developmentSlideNames;
+    $scope.developmentSlideJobTitles = developmentSlideJobTitles;
+
+    var researchSlideImages = [];
+    var researchSlideQuotes = [];
+    var researchSlideNames = [];
+    var researchSlideJobTitles = [];
+    VOLUNTEER_INFO.research.forEach(function(researcherProfile) {
+      researchSlideImages.push(researcherProfile.profilePictureImageFilename);
+      researchSlideQuotes.push(researcherProfile.professionalQuote);
+      researchSlideNames.push(researcherProfile.fullName);
+      researchSlideJobTitles.push(researcherProfile.jobTitle);
+    });
+    $scope.researchSlideQuotes = researchSlideQuotes;
+    $scope.researchSlideNames = researchSlideNames;
+    $scope.researchSlideJobTitles = researchSlideJobTitles;
+
+    var marketingSlideImages = [];
+    var marketingSlideQuotes = [];
+    var marketingSlideNames = [];
+    var marketingSlideJobTitles = [];
+    VOLUNTEER_INFO.marketing.forEach(function(marketerProfile) {
+      marketingSlideImages.push(marketerProfile.profilePictureImageFilename);
+      marketingSlideQuotes.push(marketerProfile.professionalQuote);
+      marketingSlideNames.push(marketerProfile.fullName);
+      marketingSlideJobTitles.push(marketerProfile.jobTitle);
+    });
+    $scope.marketingSlideQuotes = marketingSlideQuotes;
+    $scope.marketingSlideNames = marketingSlideNames;
+    $scope.marketingSlideJobTitles = marketingSlideJobTitles;
+
+    var createSlide = function(profileSlides, profileImage, slideId) {
+      profileSlides.push({
+        image: profileImage,
+        id: slideId
+      });
+    };
+    $scope.designSlides = [];
+    $scope.developmentSlides = [];
+    $scope.researchSlides = [];
+    $scope.marketingSlides = [];
+    // Only adding 3 profiles currently.
+    for (var i = 0; i < 3; i++) {
+      // Increase image dimension to get a diffent image per request
+      // from unsplash API
+      var newWidth = 600 + i;
+      var imageUrl = 'https://unsplash.it/' + newWidth + '/300';
```

Comment:
Actually let's not do this. For now let's upload actual pics and use that -- no external images. It's fine if they're all copies of the same pic.

The idea is that, after this PR is in, the site admin can just modify the declarative data + the pics folder and nothing else (and we should have instructions for that).

Reply:
Let me TO-DO this while I attempt to complete this PR successfully. Is that OK with you? I will have all of the images's url point to the same unsplash.it link.

Reply:
I will take care of the image in a PR where I will go around and compress all of the current images uploaded to the repo. I will then look into how to fully automate things properly.

Reply:
Can you upload one pic, and have multiple references to it? (Don't worry about compressing it.) Again the aim is to get this to a skeleton where we have an example and instructions for updating the page more generally with the data, and I think we should set a pattern for loading images from the host server (rather than externally).

OK to do this after you do the other stuff, but better to do it before submitting the PR, I think.

Reply:
Yep. I have replaced all the link with a sample image. 
![screen shot 2018-08-03 at 12 59 39 am](https://user-images.githubusercontent.com/11153258/43625016-9f362da0-96b8-11e8-9bc8-201012c71188.png)